app.js

```diff
@@ -34,23 +34,54 @@ function resetStats() {
 }
 
 form.addEventListener('submit', (e) => {
+    e.preventDefault();
     // Step 1 - add code to track each submission to the stats
-    // Create an object to "model" you statistic like the following:
+    const playerData = new FormData(form);
+    // Create an object to "model" your statistic like the following:
     // { player: 'Bob', points: 2 }
     // Hint -- create the object from the form, push it onto the stats array, then call renderStats
+    const player = {
+        player: playerData.get('player'),
+        points: playerData.get('points'),
+    };
+    stats.push(player);
+    // eslint-disable-next-line no-console
+    console.log(stats);
+    renderStats();
+    renderGames();
+    // playerData.reset();
 });
 
 remove.addEventListener('click', () => {
     // Step 2 -- add code to allow users to remove the most recent stat
     // Hint -- how do you remove an element from an array?
     // Hint -- how can we rerender the stats using a function above?
+    stats.pop();
+    renderStats();
+    renderGames();
 });
 
 save.addEventListener('click', () => {
-    // Step 3 - add code to allow users to save the state
+    // Step 3 - add code to allow users to save the stats
     // Loop through the list of stats and add up the total points scored
-    // Create a new object with the game number and the total points
-    // { number: games.length + 1, totalPoints: totalPoints }
-    // Push the new object onto the games array then call renderGames
-    // reset the stats with resetStats
+    let sum = 0;
+
+    for (let stat of stats) {
+        const numStat = Number(stat.points);
+        sum = numStat + sum;
```

Comment:
You can actually use an operator shorthand here. 

`sum += numStat` is equivalent to `sum = numStat + sum`